libs/components/ag-grid/src/lib/modules/ag-grid/header/header.component.ts

```diff
@@ -151,16 +166,16 @@ export class SkyAgGridHeaderComponent
     );
 
     this.#updateInlineHelp();
-    this.#changeDetector.markForCheck();
+    this.#changeDetector.detectChanges();
```

Comment:
Calling `detectChanges()` is [generally frowned upon](https://stackoverflow.com/a/57382013/6178885) since it can cause performance issues. If you absolutely must call `detectChanges()`, can you add a comment above to explain why?

Reply:
Good call. I reverted the change detection changes and their test still passes.